requirements-3.txt

```diff
@@ -9,8 +9,7 @@ docutils==0.15.2
 freezegun==1.2.2
 idna==2.7
 linecache2==1.0.0
-mypy==1.5.1
-mypy-extensions==1.0.0
+mypy==1.7.1
```

Comment:
I'd personally bump mypy all the way to latest. `1.7` is the minimum where `Unpack` is no longer an incomplete feature https://mypy-lang.blogspot.com/2023/11/mypy-17-released.html

Reply:
Thank you for this.  mypy is only used in this package to do the type checking so its probably good that its at the latest anyways.  That got the tests to pass now